internal/api/v1/invoice.go

```diff
@@ -341,6 +341,7 @@ func (h *InvoiceHandler) AttemptPayment(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"message": "payment processed successfully"})
 }
+
 // GetInvoicePDF godoc
 // @Summary Get PDF for an invoice
 // @Description Retrieve the PDF document for a specific invoice by its ID
```

Comment:
add the newly added param here as well so that is available when we generate docs